po/messages.pot

```diff
@@ -6,8 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: strager.nds@gmail.com\n"
-"POT-Creation-Date: 2020-10-17 14:17+0100\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"POT-Creation-Date: 2021-11-16 19:19+0100\n"
+"PO-Revision-Date: 2021-11-17 17:13+0100\n"
```

Comment:
Nit: I think `PO-Revision-Date` should not contain a real date, because this is a .pot file not a .po file. But I don't really know much about this stuff. :woman_shrugging:

Reply:
Reverted.